hack/unwanted-dependencies.json

```diff
@@ -201,13 +181,17 @@
         "sigs.k8s.io/kustomize/kustomize/v5",
         "sigs.k8s.io/kustomize/kyaml"
       ],
+      "github.com/klauspost/compress": [
+        "github.com/prometheus/client_golang",
```

Comment:
looks like this is the only caller, and intends to switch to stdlib once a zstd impl is merged

* https://github.com/prometheus/client_golang/blob/main/prometheus/promhttp/http.go#L469
* https://github.com/golang/go/issues/62513 (proposal accepted, not yet implemented)